scss/media.scss

```diff
@@ -0,0 +1,588 @@
+@media (min-width: 479px) and (max-width: 767px){
+  .header-content{
+    .h1-intro{
+      font-size: 46px;
+    }
+    h2{
+      font-size: 27px;
+    }
+  }
+
+  .another-benefits{
+    padding: 54px 60px;
+    .another-benefits-item{
+      img{
+        width: 100%;
+      }
+    }
+  }
+
+  .descr{
+    padding: 50px 60px 0;
+  }
+
+  .stuff{
+    padding: 57px 60px 65px;
+  }
+
+  .another-stuff{
+    padding: 57px 60px 65px;
+  }
```

Comment:
What's the difference between `.stuff`, `.another-stuff`? Can't you combine them?